packages/search/package.json

```diff
@@ -65,7 +65,7 @@
         "@spectrum-web-components/textfield": "^1.1.1"
     },
     "devDependencies": {
-        "@spectrum-css/search": "8.0.0-s2-foundations.18"
```

Comment:
Can you bump this to `[8.0.0-s2-foundations.19]` first and then we pull in the semver?